Mods/BreathOfTheWild_Intel Remove Bloom/rules.txt

```diff
@@ -0,0 +1,6 @@
+[Definition]
+titleIds = 00050000101C9300,00050000101C9400,00050000101C9500
+name = "Remove Intel Excess Bloom"
+path = "The Legend of Zelda: Breath of the Wild/Graphics/Intel/Remove Excess Intel Bloom"

```

Comment:
You should move this path to `The Legend of Zelda: Breath of the Wild/Workarounds/Intel iGPU/Remove Excessive Intel Bloom`.

Reply:
Path modified to
path = "The Legend of Zelda: Breath of the Wild/Workarounds/Intel iGPU/Remove Excess Intel Bloom"